### PR TITLE
reduces glare to its original state, eat shit fluffy

### PR DESCRIPTION
**Pull request by ghost**
there was no reason to buff the ability when it was already extremely powerful when it wasn't bugged

making it even more of an easy cheese tool makes shadowlings even harder to deal with, and they're already nigh impossible to deal with

:cl:  
tweak: glare's back to normal stun lengths and being two-tile ranged rather than four-tiles for an instant paralysis
/:cl:

### CODE REVIEW

**Comment by fluffe9911** [comment]
this is fine but its fluffe not fluffy lol

**Comment by Kmc2000** [comment]
i like cheese

File: yogstation/code/modules/antagonists/shadowling/shadowling_abilities.dm [hunk 1]
```diff
@@ -75,10 +75,10 @@
 	var/mob/living/carbon/human/M = target
 	usr.visible_message("<span class='warning'><b>[usr]'s eyes flash a purpleish-red!</b></span>")
```
Comment (ShadowDeath6) [inline yogstation/code/modules/antagonists/shadowling/shadowling_abilities.dm:76]: I think it’s purplish rather than purpleish

**Comment by ShadowDeath6** [comment]
Not really in the scope of this pr but I feel like glare just shouldn’t cast rather than have the stamina drain thing so shadowlings don’t end up essentially wasting their ability because of how small the effective range is

**Pull request merged**